lightning/src/ln/msgs.rs

```diff
@@ -1331,6 +1693,32 @@ impl_writeable_msg!(OpenChannel, {
 	(1, channel_type, option),
 });
 
+impl_writeable_msg!(OpenChannelV2, {
+	chain_hash,
+	temporary_channel_id,
+	funding_feerate_perkw,
+	commitment_feerate_perkw,
+	funding_satoshis,
+	dust_limit_satoshis,
+	max_htlc_value_in_flight_msat,
+	htlc_minimum_msat,
+	to_self_delay,
+	max_accepted_htlcs,
+	locktime,
+	funding_pubkey,
+	revocation_basepoint,
+	payment_basepoint,
+	delayed_payment_basepoint,
+	htlc_basepoint,
+	first_per_commitment_point,
+	second_per_commitment_point,
+	channel_flags,
+	shutdown_scriptpubkey
```

Comment:
I think from the spec this looks like it should be in the tlv stream? I might just be missing something about how `impl_writeable_msg` works but figured I'd ask